cosmic/utils.py

```diff
@@ -848,6 +848,10 @@ def error_check(BSEDict, filters=None, convergence=None, sampling=None):
     if flag in BSEDict.keys():
         if BSEDict[flag] <= 0:
             raise ValueError("'{0:s}' needs to be greater than 0 (you set it to '{1:0.2f}')".format(flag, BSEDict[flag]))
+    flag='rembar_massloss'
```

Comment:
I think this is currently breaking the tests

Reply:
hmmm, why would this break them? any suggestions for how to  fix?

Reply:
you can see how they broke here: https://travis-ci.com/github/COSMIC-PopSynth/COSMIC/jobs/320663446

it looks like the test_single_evolve_with_inifile and test_multi_evolve_with_inifile are having some trouble with the inifile. I think @scottcoughlin2014 wrote these tests? Maybe they just need to be updated? It looks like the indices are hard coded in the test.